Schedule meetups through November 2018

dasanchez opened the pull request:
FreeGeek and HackLab will host the following events:
- Aug 23: Meet and Greet @ FreeGeek
- Sep 20: Meet and Greet @ HackLab
- Oct 25: Meet and Greet @ FreeGeek
- Nov 22: Planning Session @ HackLab

dasanchez committed:
Schedule meetups through November 2018

dasanchez committed:
Update 2018-09-20-meet-n-greet.md

Added dashes to slug text

Shrinks99 commented:
Should the planning meeting specify that it is mainly aimed at current members?  While I know it's always the last one up and then we don't plan anything else until it's finished I think it would be good to direct new people to the meet & greet sessions instead.

dasanchez commented:
@Shrinks99 I would like it if people attending the planning session were aware of recent activities and had some context to contribute and feel like they belong. If a planning session is your first time attending a meetup, you might be a bit lost and require some explanation.

That said, I like the idea of being inclusive and not having an "invite-only" club. I'm OK with rewording the last statement to suggest newcomers attend a monthly meetup first:

> New to Toronto Mesh? Join the chat or visit a monthly meetup to meet the group and become familiar with ongoing activities!

Shrinks99 commented:
I like rewording the last statement to make that more clear.  

While I don't want Toronto Mesh to be an invite only club by any means I wouldn't be necessarily opposed to restricting planning meetings to existing folks (those who are on the chat and do things?  We don't need to have anything super strict like "you can't come here" style or any nonsense like that) as I don't think the planning meetings are a particularly great introduction for the users that find our website on the internet, see that there's a meeting coming up and go to it to meet people and see whats up.  They're an hour long and full of information that is useful to pretty much nobody who is new and just wants to learn about what mesh networking is.

Of course this needs to be brought up with more people, maybe it's something to address at the planning meeting? :P

dasanchez committed:
Changed the message for newcomers

Planning sessions might not be the best place to start for people who simply want to learn about mesh- the message suggests newcomers attend a monthly meetup instead.

dasanchez commented:
Thanks @Shrinks99! I updated the closing message for the planning session.

benhylau commented on _posts/2018-11-22-planning-session.md:
```diff
@@ -0,0 +1,26 @@
+- Set the priorities and projects to work on for the next six months.
+- Assign working group leads for the next six months. You can see our current [working groups and governance processes here](https://github.com/tomeshnet/documents/blob/master/governance/coordination-structure.md)!
+
+
```
Random newline?

dasanchez replied:
Whoops- fixed.

benhylau commented on _posts/2018-10-25-meet-n-greet.md:
```diff
@@ -0,0 +1,17 @@
+date: 2018-10-25
+startTime: '19:00'
+endTime: '21:00'
+---
```
We should tag this to all our events:

>This event has a [code of conduct](/code-of-conduct/).

dasanchez replied:
All events are getting the Code of Conduct line now.

dasanchez committed:
Added code of conduct to Meet and Greet events

garrying commented on _posts/2018-11-22-planning-session.md:
```diff
@@ -0,0 +1,25 @@
+endTime: '21:30'
+---
+
+This us Toronto Mesh's seventh planning session. We will regroup and plan for the winter and beyond! During this session we will:
```
`This us` -> `This is`?

benhylau replied:
Good catch

garrying commented on _posts/2018-11-22-planning-session.md:
```diff
@@ -0,0 +1,25 @@
+
+This event has a [code of conduct](/code-of-conduct/).
+
+New to Toronto Mesh?  
```
Maybe this could be a heading or bolded?

dasanchez replied:
Agreed! I'll update it.

dasanchez committed:
Fixed typo and emphasized newcomer message

dasanchez commented:
Thanks for the comments @benhylau and @garrying! I updated the post accordingly.
